test_example_conversions.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
Either leave off the shebang or invoke [`pytest.main`](http://docs.pytest.org/en/latest/usage.html#calling-pytest-from-python-code) if `__name__ = '__main__'.